src/devices/RealtimeClock/Devices/Ds1307/Ds1307Register.cs

```diff
@@ -2,12 +2,12 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-namespace Iot.Device.Ds1307
+namespace Iot.Device.Rtc
```

Comment:
I like this. We should do similar stuff for Mcp3008 where Mcp3008 is namespace and class name which causes it to be really annoying to use as you always have to type both (not in this PR)